extensions/amp-story/1.0/bookend/components/article.js

```diff
@@ -101,15 +101,16 @@ export class ArticleComponent {
     }
 
     if (articleData.image) {
-      const ampImg =
+      const imgEl =
           html`
-          <amp-img class="i-amphtml-story-bookend-article-image"
-                  width="100"
-                  height="100">
-          </amp-img>`;
-
-      addAttributesToElement(ampImg, dict({'src': articleData.image}));
-      el.insertBefore(ampImg, el.firstChild);
+          <div class="i-amphtml-story-bookend-article-image">
```

Comment:
Here and in the other templates: Can we get rid of this (unnecessary?) div?

Reply:
We need it make the images take the full width and height of its available space. Also to do the `padding-bottom` css trick on the portrait and landscape cards to keep a width and expand/decrease the height.

Reply:
Perfect, thanks :)